src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static String getRandomBall() {
```

Comment:
1. same about static
2. this method should return Ball
```suggestion
    public Ball getRandomBall() {
```